src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.de.xlf

```diff
@@ -89,7 +89,7 @@
       </trans-unit>
       <trans-unit id="DefineDiagnosticTitleCorrectlyMessage">
         <source>The diagnostic title should not contain a period, nor any line return character, nor any leading or trailing whitespaces</source>
-        <target state="translated">Der Diagnosetitel darf keinen Satzendepunkt, kein Zeilenvorschubzeichen, keine führenden und keine nachfolgenden Leerzeichen enthalten.</target>
+        <target state="translated">Der Diagnosetitel darf weder einen Punkt, noch ein Zeilenumbruchzeichen, noch führende und nachfolgende Leerzeichen enthalten.</target>
```

Comment:
This is the only translated resource in this file, while other files have more translated resources. I picked a random resource "PlatformCompatibilityAllVersions", and I couldn't find it completely in this resource file. Most likely, this will cause the message to default to English as far as I know.

Note that "PlatformCompatibilityAllVersions" is part of other message, so it could happen that part of the diagnostic will appear in the correct language, while another part will be English.

The missing resource (or resources?) may need investigation, it should have failed the build unless I'm missing something.

@mavasani Any idea?